### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by dblBot**

### CODE REVIEW

File: locales/de/translation.json [hunk 1]
```diff
@@ -144,6 +142,7 @@
   "Save": "Speichern",
   "Sending...": "Sendet...",
   "Serial Number": "Seriennummer",
+  "Settings on day": "Einstellungen am {{Tag}}",
```
Comment (celia-rouis) [inline locales/de/translation.json:145]:
```suggestion
  "Settings on day": "Einstellungen am {{day}}",
```

File: locales/de/yourloops.json [hunk 2]
```diff
@@ -58,20 +61,23 @@
   "alarm-kaleido-empty-pump-battery-title": "Leere Pumpen-Batterie",
   "alarm-kaleido-insulin-cartridge-expired-description": "Ihre Pumpe ist stehen geblieben, weil die Insulinkartusche abgelaufen ist.",
   "alarm-kaleido-occlusion-description": "Die Pumpe ist gestoppt, weil sie eine Verstopfung festgestellt hat.",
-  "alarm-long-hyperglycemia-description": "Die G6-Werte sind seit mehr als 20 Minuten höher als {{value}} {{unit}}.",
+  "alarm-long-hyperglycemia-description-new": "Hyperglykämie während eines Zeitraums",
+  "alarm-long-hyperglycemia-description-old": "Die G6-Werte sind seit mehr als 20 Minuten höher als {{Wert}} {{Einheit}}.",
```
Comment (celia-rouis) [inline locales/de/yourloops.json:65]:
```suggestion
  "alarm-long-hyperglycemia-description-old": "Die G6-Werte sind seit mehr als 20 Minuten höher als {{value}} {{unit}}.",
```

File: locales/fr/translation.json [hunk 3]
```diff
@@ -144,6 +142,7 @@
   "Save": "Sauvegarder",
   "Sending...": "Envoi...",
   "Serial Number": "Numéro de série",
+  "Settings on day": "Paramètres sur {{day}}",
```
Comment (celia-rouis) [inline locales/fr/translation.json:145]: Not consistent with the English version

image

File: locales/fr/yourloops.json [hunk 4]
```diff
@@ -471,6 +476,10 @@
   "severe-hypoglycemia-threshold": "Seuil d'hypoglycémie sévère",
   "show-column": "Afficher les colonnes",
   "signup": "S'inscrire",
+  "signup-information": "Informations d'inscription",
+  "signup-information-message-1": "Vous êtes sur le point de créer un compte de professionnel de la santé ou d'aidant.",
+  "signup-information-message-2": "<strong>Si vous êtes un patient</strong>, votre compte sera créé quand vous initialiserez votre DBL.. <strong> Vous ne devez pas créer de compte ici.</strong>",
```
Comment (celia-rouis) [inline locales/fr/yourloops.json:481]: There should not be `..` at the end of the sentence?

Comment (celia-rouis) [reply locales/fr/yourloops.json:481]:
```suggestion
  "signup-information-message-2": "<strong>Si vous êtes un patient</strong>, votre compte sera créé quand vous initialiserez votre DBL. <strong>Vous ne devez pas créer de compte ici.</strong>",
```

**Review by TimCanu — APPROVED** [review]

**Pull request merged**